cmd/fleetctl/fleetctl.go

```diff
@@ -26,7 +26,10 @@ func init() {
 
 func main() {
 	app := createApp(os.Stdin, os.Stdout, exitErrHandler)
-	app.Run(os.Args)
+	if err := app.Run(os.Args); err != nil {
```

Comment:
Currently, missing required flags errors do not get printed and `fleetctl` exits with a status code of 0. The help text does get printed, but the exit handler does not. Imo, this is strange and unexpected behaviour. 

I believe the relevant code is [here](https://github.com/urfave/cli/blob/v2.23.5/command.go#L207).

```
	cerr := cCtx.checkRequiredFlags(c.Flags)
	if cerr != nil {
		_ = ShowSubcommandHelp(cCtx)
		return cerr
	}
```

If `cerr` is not nil, meaning that there is a missing required flag, then the help text is printed, but instead of calling the `*App.ExitErrHandler`, the error is returned by `*App.Run`.